src/NServiceBus.Core/Routing/RoutingFeature.cs

```diff
@@ -87,5 +96,35 @@ protected internal override void Setup(FeatureConfigurationContext context)
                 }
             }
         }
+
+        void EnableBestPracticeEnforcement(FeatureConfigurationContext context)
+        {
+            var validations = new Validations(context.Settings.Get<Conventions>());
+
+            context.Pipeline.Register(
+                "EnforceSendBestPractices",
+                new EnforceSendBestPracticesBehavior(validations),
+                "Enforces send messaging best practices");
+
+            context.Pipeline.Register(
+                "EnforceReplyBestPractices",
+                new EnforceReplyBestPracticesBehavior(validations),
+                "Enforces reply messaging best practices");
+
+            context.Pipeline.Register(
+                "EnforcePublishBestPractices",
+                new EnforcePublishBestPracticesBehavior(validations),
+                "Enforces publish messaging best practices");
+
+            context.Pipeline.Register(
```

Comment:
Yes, the validation in routing config apply only to unicast transports which actually require subscriber-side configuration. On brokers you only call `Subscribe.